lib/src/TailwindComponents/ButtonPureCss/button.module.css

```diff
@@ -0,0 +1,309 @@
+@import 'tailwindcss';
+
+/* The 'components' layer is used so that the styles can be overridden by Tailwind's 'utilities' layer */
+@layer components {
+  .root {
+    position: relative;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    cursor: pointer;
+    border-style: solid;
+
+    /* VARIABLES FROM THEME - DONT CHANGE WITH REACT PROPS */
+    border-width: var(--border-width-sm);
+    transition-property: all;
+    transition-duration: var(--duration-medium);
+    outline: transparent solid var(--border-width-md);
+    font-weight: var(--font-weight-bold);
+    border-radius: var(--radius-md);
+
+    &:focus {
+      outline-color: var(--outlineColorFocus);
+    }
+
+    &:hover {
+      background-color: var(--color-brand-30);
+      border-color: var(--color-brand-30);
+    }
+
+    &:active {
+      background-color: var(--color-brand-10);
+      border-color: var(--color-brand-10);
+    }
+
+    &:disabled,
+    &[aria-disabled] {
+      background-color: var(--color-beige-40);
+      border-color: var(--color-beige-40);
+      color: var(--color-beige-70);
+      outline-color: var(--color-beige-10);
+      cursor: not-allowed;
+
+      &:hover,
+      &:focus,
+      &:active {
+        background-color: var(--color-beige-40);
+        border-color: var(--color-beige-40);
+        color: var(--color-beige-70);
+        outline-color: var(--color-beige-10);
+      }
+    }
+
+    &:has(i[data-wui-icon-font]),
```

Comment:
💭 Quick question: since :has() is supported by ~93% of browsers, do we have a team policy on using it (progressive enhancement vs. full compatibility)?

Reply:
It seems that it has the same compatibility as what we currently also accept regarding our javascript ecosystem, so it looks ok to me.